gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Modal bootstrap

### Description

Made the Popup work with Bootstrap. Previously it was working on Materialize.

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
```

1. Create a match between 2 users marking them as interested/going to an event.
2. You should see a popup like the previous sprint. Message button closes the popup right now.

### Miscellaneous Comments



### Checklist
